src/components/actions/evaluatorFunctions.ts

```diff
@@ -5,7 +5,10 @@ import { DateTime, Duration } from 'luxon'
 
 const generateExpiry = (duration: Duration) => DateTime.now().plus(duration).toJSDate()
 
-const getYear = () => new Date().getFullYear()
+// getYear() => "2022"
+// getYear("short") => "22"
+const getYear = (type?: 'short'): string =>
+  type === 'short' ? String(new Date().getFullYear()).slice(2) : String(new Date().getFullYear())
```

Comment:
Off-track, but needed for one of the demo configs

Reply:
ohh that's where it's added, thanks